scripts/coverage.sh

```diff
@@ -9,21 +9,17 @@ PROJECT_DIR=$(pwd -P)
 COVERRAGE_DIR="$PROJECT_DIR"/coverage
 COVERAGE_FILE="$PROJECT_DIR"/coverage/lcov.info
 BIN_DIR="$PROJECT_DIR"/node_modules/.bin
-TEST_GLOB=dist/test/**/*-test.coffee
+TEST_GLOB="$PROJECT_DIR"/dist/test/**/*-test.js
 
 
 # Cleanup & preparation
 rm -rf "$COVERRAGE_DIR"
 rm -rf dist
-npm run copy:test
-npm run build
+npm run make
 
 
 # Testing
 export COVERAGE_FILE
-istanbul cover ./node_modules/.bin/mocha -- \
-    "$TEST_GLOB" \
-    --compilers=coffee:coffee-script/register \
-    --recursive
+istanbul cover ./node_modules/.bin/_mocha -- "$TEST_GLOB" --opts=./mocha.opts
```

Comment:
In npm scripts it seems using just `dist/test` is sufficient and no glob is needed.

Reply:
Didn't we want to move the opts back to the test directory? Then we could omit the `--opts` option here and in package.json.

Reply:
Hmm... Correct me if I'm wrong, but this file could be now completely rewritten as an npm script:

```
"coverage": "rm -rf ./coverage && npm run build && istanbul cover _mocha -- dist/test"
```

And I'm not completely sure whether `rm -rf ./coverage` is necessary.

Reply:
Addressed in https://github.com/apiaryio/dredd-transactions/pull/128/commits/e5f0e92e37a9f86c4d73d08d58fa5cf197387f70 and https://github.com/apiaryio/dredd-transactions/pull/128/commits/1755b818097d45525c052e264a420a3782bfeb96.